docs/guides/testing/testing.rst

```diff
@@ -117,143 +117,67 @@ Note -
 
     paver -h
 
+Note -
+Unless otherwise mentioned, all the following commands should be run from inside lms docker container.
 
 Running Python Unit tests
 -------------------------
 
-We use `pytest`_ to run the test suite.
+We use `pytest`_ to run python tests. Pytest is a testing framework for python and should be your goto for local python unit testing.
 
 .. _pytest: https://pytest.org/
 
-For example, this command runs all the python test scripts::
-
-    paver test_python
-
-It also runs ``collectstatic``, which prepares the
-static files used by the site (for example, compiling Sass to
-CSS).
-
-You can re-run all failed python tests by running this command (see note at end of
-section)::
-
-    paver test_python --failed
-
-To test lms python tests use this command::
-
-    paver test_system -s lms
-
-To test cms python tests use this command::
-
-    paver test_system -s cms
-
-To run these tests without ``collectstatic``, which is faster, append the following argument::
-
-    paver test_system -s lms --fasttest
-
-To run cms python tests without ``collectstatic`` use this command::
-
-    paver test_system -s cms --fasttest
-
-For the sake of speed, by default the python unit test database tables
-are created directly from apps' models. If you want to run the tests
-against a database created by applying the migrations instead, use the
-``--enable-migrations`` option::
-
-    paver test_system -s lms --enable-migrations
-
-To see the migration output, use::
-
-    paver test_system -s lms --enable-migrations --verbose --disable_capture
-
-To run a single django test class use this command::
-
-    paver test_system -t lms/djangoapps/courseware/tests/tests.py::ActivateLoginTest
 
 Running a Single Test
 ~~~~~~~~~~~~~~~~~~~~~
 
-When developing tests, it is often helpful to be able to really just run
-one single test without the overhead of PIP installs, UX builds, etc. In
```

Comment:
Nit: I would actually keep this first sentence, since the context is applicable to `pytest` as much as it was `paver`.